src/arche/arche.py

```diff
@@ -38,8 +38,13 @@ def __init__(
             start: an item key to start reading from
             filters: Scrapinghub filtering, see
             https://python-scrapinghub.readthedocs.io/en/latest/client/apidocs.html#scrapinghub.client.items.Items # noqa
-            expand: if True, use flattened data in garbage rules
         """
+        if expand:
+            maintenance.deprecate(
+                "'expand' parameter is deprecated and will be removed in the next 0.3.6"
+                " release. See CHANGES.md for more details.",
```

Comment:
Since `CHANGES.md` does not seem to provide any additional detail (no explanation as to why it was removed, or suggestion of an alternative), wouldn’t it be better not to mention `CHANGES.md` here at all?

Reply:
I gave more details, and it's easy backtrack to this pr from `changes.md`.